src/core/ddsi/include/dds/ddsi/shm_init.h

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
Note that the file might get a more general name later. The idea is to collect everything needed for setting up and shutting down shared memory support here (i.e. init and destruction) as well as some helper methods. Monitoring shared memory activity (i.e. running while using it) is done in sh,_monitor.h, however.